### PR TITLE
feat(core): Initial structure

### PR DESCRIPTION
**Pull request by pavlovicnemanja**
TODO:
- [x] Added initial structure from angular2-seed
- [x] Add tslint task
- [x] Add karma task
- [x] Add sass loader

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        __dirname: true,
+        __filename: true,
+        process: true,
+        Buffer: false,
+        clearImmediate: false,
+        setImmediate: false
+    }
+};
+
+
```
Comment (takeit) [inline webpack.config.js]: new line not needed

**Review by takeit — COMMENTED** [review]
new line at the end of each file is missing

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/nbproject
```
Comment (ahilles107) [inline .gitignore]: it should got to your local gitignore (I assume that directory is connected to your editor)

Comment (pavlovicnemanja) [reply .gitignore]: then I will need to exclude nbproject folder every time I make a PR... not sure why to remove it, this is only developer thing :smile:

Comment (ahilles107) [reply .gitignore]: Hey, just add it to Your global gitignore and you will not need to ignore it anymore in any other repository. Read more here: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Comment (pavlovicnemanja) [reply .gitignore]: tnx :smile:

**Review by ahilles107 — APPROVED** [review]

**Review by takeit — APPROVED** [review]

**Pull request merged**